### PR TITLE
Podify

### PR DESCRIPTION
**Pull request by keithburgoyne**
Turn this package into a CocoaPod.

### CODE REVIEW

File: JREProgressView.podspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  s.authors      =  { "Keith Burgoyne"   => "keith@silverorange.com",
+                      "Michael Gauthier" => "mike@silverorange.com",
+                      "John Rommel Estropia" => "" }
+  s.platform     = :ios, "7.1"
+  s.ios.deployment_target = "7.1"
+  s.source       = { :git => "https://github.com/silverorange/JREProgressView.git", :tag => s.version.to_s }
+  s.source_files = "JREProgressView/*.{h,m}"
+  s.public_header_files = "JREProgressView/*.h"
+  s.requires_arc = true
+  s.framework    = "UIKit"
```
Comment (gauthierm) [inline JREProgressView.podspec]: should be `s.frameworks`

File: JREProgressView.podspec [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                   UIProgressView subclass that supports custom images on iOS 7.1.
+                   DESC
+  s.homepage     = "https://github.com/silverorange/JREProgressView"
+  s.license      = { :type => "MIT", :file => "LICENSE" }
+  s.authors      =  { "Keith Burgoyne"   => "keith@silverorange.com",
+                      "Michael Gauthier" => "mike@silverorange.com",
+                      "John Rommel Estropia" => "" }
+  s.platform     = :ios, "7.1"
+  s.ios.deployment_target = "7.1"
+  s.source       = { :git => "https://github.com/silverorange/JREProgressView.git", :tag => s.version.to_s }
```
Comment (gauthierm) [inline JREProgressView.podspec]: `https://` github URLs will require username/password auth that will not work with 3rd-party auth enabled Github accounts. Use the SSH git URL instead.

Comment (gauthierm) [inline JREProgressView.podspec]: This package is public, so the above comment does not apply.

Comment (keithburgoyne) [inline JREProgressView.podspec:15]: We decided to use SSH for private packages and HTTPS with public.

**Pull request merged**